modules/aioseop_sitemap.php

```diff
@@ -4284,6 +4373,87 @@ public function get_all_post_priority_data( $include = 'any', $status = 'publish
 
 			$links = $this->get_prio_from_posts( $posts, $this->get_default_priority( 'post', true ), $this->get_default_frequency( 'post', true ) );
 			$links = array_merge( $links, $this->get_archive_prio_from_posts( $posts ) );
+
+			$is_sitemap_indexes_disabled = empty( $this->options['aiosp_sitemap_indexes'] );
+			if ( $is_sitemap_indexes_disabled || ( ! $is_sitemap_indexes_disabled && 'page' === $include ) ) {
```

Comment:
The `$include` variable can also be an array. Might want to also add `|| in_array( 'page', $include )`...but I suppose `$is_sitemap_indexes_disabled` usually handles multiple $includes.

Which would also mean $include isn't always a string in the phpDocs, and wasn't updated sometime in the past.

Reply:
That won't be needed since, like you've said yourself, sitemap indexes only contain one type of post (or taxonomy).